integration-tests/chopsticks/src/network/spiritnet.ts

```diff
@@ -0,0 +1,60 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import * as HydraDxConfig from './hydroDx.js'
+
+export const options: SetupOption = {
+	endpoint: 'wss://kilt-rpc.dwellir.com',
+	db: './db/spiritnet.db.sqlite',
+	port: 8002,
+	wasmOverride: '../../target/debug/wbuild/spiritnet-runtime/spiritnet_runtime.wasm',
+	allowUnresolvedImports: true,
+}
+
+export const defaultStorage = (addr: string) => ({
+	technicalCommittee: { Members: [addr] },
+	council: { Members: [addr] },
+	System: {
+		Account: [[[addr], { providers: 1, data: { free: 1000 * 1e12 } }]],
+	},
+	polkadotXcm: {
+		safeXcmVersion: 3,
+	},
+})
+
+export const paraId = 2086
+export const hydraDxDestinationV2 = {
+	V2: {
+		parents: 1,
+		interior: {
+			X1: {
+				Parachain: HydraDxConfig.paraId,
+			},
+		},
+	},
+}
+export const hydraDxBeneficiaryV2 = {
+	V2: {
+		parents: 1,
```

Comment:
This does not match what we were told by HydraDX guys. `parents` should be `0`.

Reply:
And that makes sense since this is evaluated already on HydraDX, why would there be a parent in here?

Reply:
Yes, that was a bug. It is resolved, but I can not find the commit.